Net07.HW/Net07.DynamicProgrammingAndClasses/Net07.DynamicProgrammingAndClasses/Shape.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Net07.DynamicProgrammingAndClasses
+{
+    class Shape
+    {
+        public double Area { get; }
```

Comment:
In one of the shapes, we can forget to initialize `Area` and `Perimeter`.
 We can make `Area` and `Perimeter` abstract to avoid this